### PR TITLE
Feature: Show "Transfer" menu item to user when transfer bar is dismissed

### PR DESCRIPTION
**Pull request by alaca**
## Description
After a user migrates a v2 form to a v3 form they're shown the builder. At the bottom of the builder is a blue bar telling them how to transfer data. If they dismiss that bar, a "Transfer Donation Data"  menu item shows up in the site options.

This PR is adding functionality that will open the sidebar menu and show the user the menu item so they know how to transfer when they're ready.

## Visuals

Screen Capture_select-area_20230919202339

## Testing Instructions

- Add form
- Upgrade the form
- Dismiss the transfer bottom bar

When the transfer bottom bar is dismissed, the sidebar menu should be opened and the Transfer Donation Data menu item should be selected. Also, the tooltip which explains what that menu item does should be visible (see attached video)

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205502412661044
  - https://app.asana.com/0/0/1205550233964861

### CODE REVIEW

File: src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx [hunk 1]
```diff
@@ -132,15 +132,31 @@ const HeaderContainer = ({
...
+                                : __('Publish', 'give')}
                     </Button>
                     <Button onClick={toggleShowSidebar} isPressed={showSidebar} icon={drawerRight} />
                     <Dropdown
                         popoverProps={{placement: 'bottom-start'}}
                         // @ts-ignore
                         focusOnMount={'container'}
-                        renderToggle={({isOpen, onToggle}) => <Button onClick={onToggle} icon={moreVertical} />}
+                        renderToggle={({isOpen, onToggle}) => {
+                            if (!isOpen && transfer.showTooltip) {
```
[the remark this answers is not in the document]
Comment (alaca) [reply src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx]: Good point. Resolved https://github.com/impress-org/givewp/pull/6954/commits/d717f3bebed45fe8588d5650dd0388be85b1c710

**Review by kjohnson — APPROVED** [review]
Thanks for that change! Let's get this to QA.

**Comment by kjohnson** [comment]
Building for QA:
- https://github.com/impress-org/givewp/actions/runs/6264722362
- https://app.asana.com/0/1203193833018151/1205550233964861/f

**Review by rickalday — APPROVED** [review]
Passed manual QA tests.

**Pull request merged**